### DISCUSSION TITLE
6900 XT recognized as 6800 Series + little mh/s drops after few min.

### DISCUSSION THREAD

**Discussion by t1mstar** [Lolliedieb/lolMiner-releases#363] [Q&A]
Hello,
Im a newbie and just started with mining.
lolMiner recognize my 6900 XT as 6800 Series. I think I dont have to care about it because it probably not effecting the performance?
![card 6800 instead of 6900](https://user-images.githubusercontent.com/33476753/111164256-31b1e080-859e-11eb-9cf9-da4daf0a0e68.png)

But what I care about, is that the performance of my 6900 XT drops ~1.5 mh/s after few min. I know its not that much, but you want the best performance you can get ofc.
So the weird thing is, its reproducable. If I restart lolminer I get 63 mh/s in the beginning and few min. I get 61.5 mh/s.
Any Ideas?
And one more thing is: why I have GPU:0 and GPU:1 and in the beginning I have only stats @ GPU:0 and after few min. it will be splitted GPU:0/GPU:1.

![63mhs](https://user-images.githubusercontent.com/33476753/111164775-aab13800-859e-11eb-92a7-e16c2f51d6ee.png)
![61mhs](https://user-images.githubusercontent.com/33476753/111164769-a8e77480-859e-11eb-9aa6-0a5fede79e6a.png)

**Comment by ben219brown** [comment]
Miner is treating your card as 2 devices.

Edit your bat file so it reads lolminer.exe --devices 0 <rest of commands>

Then you will only mine on GPU 0 and have a hashrate closer to 64 consistently

**Reply by ben219brown** [reply]
Example

```
:loop
lolMiner.exe --devices 0 --algo ETHASH --pool !POOL! --user !WALLET! --watchdog exit
if %ERRORLEVEL% == 42 (
	timeout 10
	goto loop
)
```
